client/src/components/teams/NewTeamModal.vue

```diff
@@ -55,7 +55,38 @@ const form = ref({
 const submitForm = () => {
   loading.value = true
   teamStore.createTeam({ ...form.value }).catch((error: AxiosError | any) => {
-    if (error instanceof AxiosError && error.response) {
+    console.log(error.response.data.name)
+    if (
+      error instanceof AxiosError &&
+      error.response !== undefined &&
+      error.response.data.name == 'team with this name already exists.'
```

Comment:
This should start with a capital like the other error messages.

Reply:
that won't work, thats how error.response defines that error message

Reply:
replacing it would just make it come up with a generic error message

Reply:
Oh, you are correct, sorry about that

Reply:
all g